app/classes/site_data.rb

```diff
@@ -35,8 +35,7 @@
 #      },
 #    }
 #
-################################################################################
-
+#
 class SiteData
```

Comment:
Half-baked idea: I don't have time to do it now, but I wonder if this old class can be refactored by replacing the sql queries with chained ActiveRecord scopes.